### PR TITLE
i18n(es): Update `astro-components.md`, `layouts.md` and `astro-pages.md`

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- update `astro-components.md` to match with latest changes in #787 
- update `layouts.md` to match with latest changes in #917 
- update `astro-pages.md` to match with latest changes in #895

### CODE REVIEW

File: src/pages/es/core-concepts/astro-components.md [hunk 1]
```diff
@@ -106,20 +106,20 @@ const misPokemonesFavoritos = [/* ... */];
...
+### Expresiones JSX
 
-Los componentes de Astro pueden definir variables locales dentro del script del componente. Cualquiera de estas variables van a estar disponibles para ser utilizadas en el maquetado del componente de manera automática.
+Puedes definir variables locales de JavaScript dentro del script del componente de Astro. ¡Luego puedes inyectar estas variables en el maquetado del componente usando expresiones JSX!
 
-#### Valores dinámicos
+#### Variables
 
-Las variables locales pueden ser utilizadas entre llaves, en este caso su valor va a ser utilizado en el HTML generado:
+Las variables locales pueden ser agregaradas al maquetado usando la sintaxis de llaves:
```
Comment (agustinmulet) [inline src/pages/es/core-concepts/astro-components.md:115]:
```suggestion
Las variables locales pueden ser agregadas al maquetado usando la sintaxis de llaves:
```

File: src/pages/es/core-concepts/astro-pages.md [hunk 2]
```diff
@@ -45,20 +45,20 @@ import MySiteLayout from '../layouts/MySiteLayout.astro';
 </MySiteLayout>
 ```
 
-📚 Lea más sobre [componentes plantilla](/es/core-concepts/layouts/) en Astro.
+📚 Lea más sobre [componentes de plantilla](/es/core-concepts/layouts/) en Astro.
```
Comment (agustinmulet) [inline src/pages/es/core-concepts/astro-pages.md:48]:
```suggestion
📚 Lee más sobre [componentes de plantilla](/es/core-concepts/layouts/) en Astro.
```

File: src/pages/es/core-concepts/layouts.md [hunk 3]
```diff
@@ -48,39 +52,55 @@ import MySiteLayout from '../layouts/MySiteLayout.astro';
...
+## Plantillas anidadas
 
-Cuando una página de Markdown utiliza una plantilla, se le pasa a la plantilla un elemento `content` el cual incluye todos los metadatos del Markdown y el HTML final renderizado. Consulte el ejemplo anterior de [`BlogPostLayout.astro`](/es/core-concepts/layouts/#plantilla-de-ejemplo) para ver cómo usaría `content` en su componente plantilla.
+Los componentes de plantilla no necesitan contener una página completa de HTML. Puedes dividir tus plantillas en componentes más pequeños y luego reutilizar estos componentes para crear plantillas aún más flexibles y potentes en tu proyecto.
 
-```markdown
+Por ejemplo, una plantilla común para artículos de blog suele contener un título, fecha y autor. El componente de plantilla `BlogPostLayout.astro` puede agregar esta UI, también puedes utilizar una plantilla más grande para todo el sitio web que maneje el resto de la página.
```
Comment (agustinmulet) [inline src/pages/es/core-concepts/layouts.md:92]:
```suggestion
Por ejemplo, una plantilla común para artículos de blog suele contener un título, fecha y autor. El componente de plantilla `BlogPostLayout.astro` puede agregar esta UI y también puedes utilizar una plantilla más grande para todo el sitio web que maneje el resto de la página.
```

**Review by agustinmulet — COMMENTED** [review]
Just some little suggestions

**Comment by agustinmulet** [comment]
LGTM! ✅ Fantastic work!

**Comment by Yan-Thomas** [comment]
Thanks @kevinzunigacuellar and @agustinmulet, great work!! The Spanish team is on fire!! 🚀

**Pull request merged**